### PR TITLE
Tv button

### PR DESCRIPTION
**Pull request by mariaalouisaa**
**Imported GIF & added button CSS.**

Couple things to note:
- I tested the button on the screen reader, and the only way I could get it to recognise the button as a button was to give the image `aria-hidden="true"` so that's what that's about!
- We will probably want to add some media queries once we know the width of our list, to make the button smaller at a certain breakpoint.

Also, could be fun to add CSS transition to the button on hover. 

Let me know if you have any questions about the code/requests for any changes!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,10 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Just+Another+Hand&display=swap" rel="stylesheet">
+    <link
```
Comment (PJSalter) [inline index.html:7]: Clearly neatly presented on the page, this is great ⭐

File: index.html [hunk 2]
```diff
@@ -14,16 +17,16 @@ <h1>Binge Pad</h1>
     </header>
     <!-- background -->
     <div class="no-signal">
-    
-   <!-- button now within container of background styling -->
-    <button id="new-list" class="tv">FUNKY TV IMAGE OR GIF!</button>
-  
+      <button aria-label="Add a new list" id="new-list" class="tv">
+        <img src="tv-gif.gif" aria-hidden="true" alt="Gif of plasticine TV with dancing antennas"/>
```
Comment (PJSalter) [inline index.html:21]: I love the dancing Antennas 🤩  they are great 📺  brilliant work.

so I was just looking up the (aria-hidden="true") it's super interesting so this improves the experience for accessibility by hiding anything that is decorative or images, icons or repeated colour effects, I see thats a great way to making the whole button/tv-gif visible 🙌  

Great work ✨

File: style.css [hunk 3]
```diff
@@ -55,11 +55,34 @@ h1 {
...
+  z-index: 5;
+}
+
+.tv p {
+  /* position relaive used to move the <p> up so it appears in the TV */
+  position: relative;
+  bottom: 7.7em;
+  left: 1.7em;
+  width: 3em;
+  color: #EFD30B;
```
Comment (PJSalter) [inline style.css:80]: Yellow stands out greatly with the black background screen, good choice 🎨 

maybe we could use the CSS variable from the root colours that i'll show soon to keep consistency ?

Great work 💯

File: style.css [hunk 4]
```diff
@@ -55,11 +55,34 @@ h1 {
...
+  background-color: black;
+  padding: 1em;
+  z-index: -1; /* I'm not sure if it's good practice to use negative z-index but this is the only way to get the text behind the tv */
+  letter-spacing: 2px;
```
Comment (PJSalter) [inline style.css:84]: I noticed that you like to use 'em' a lot of the time, possibly keeping it consistent with 'em' would be great and changing the 'px' however this is only a minor thing.

it's a good amount of letter spacing to show graphically on the tv screen. ✨

File: style.css [hunk 5]
```diff
@@ -55,11 +55,34 @@ h1 {
...
+}
+
+.tv p {
+  /* position relaive used to move the <p> up so it appears in the TV */
+  position: relative;
+  bottom: 7.7em;
+  left: 1.7em;
+  width: 3em;
+  color: #EFD30B;
+  background-color: black;
```
Comment (PJSalter) [inline style.css:81]: Same as previous review comment with the use of CSS Variable later on 💫

Comment (PJSalter) [reply style.css:81]: Great choice in background colour, this will display the bright text like a chalk board 🎒

File: style.css [hunk 6]
```diff
@@ -55,11 +55,34 @@ h1 {
...
+  border: none;
+  height: 10em;
+}
+
+.tv img {
+  width: 10em;
+  z-index: 5;
+}
+
+.tv p {
```
Comment (PJSalter) [inline style.css:74]: very neat CSS Styling, clearly readable 🙌 🐰 ✨

**Review by PJSalter — APPROVED** [review]

File: style.css [hunk 7]
```diff
@@ -55,11 +55,34 @@ h1 {
   color: #EFD30B
```
Comment (AsmahanM) [inline style.css:55]: Update please.

File: style.css (hunk 3 above)
Comment (AsmahanM) [inline style.css:80]: Update please.

**Review by AsmahanM — APPROVED** [review]

**Pull request merged**